vignettes/lintr.Rmd

```diff
@@ -93,8 +93,9 @@ The first one found, if any, will be used.
 
 1.  If `options(lintr.linter_file)` is an absolute path, this file will be used. The default for this option is `".lintr"`.
 2.  A linter file (that is a file named like `lintr.linter_file`) in the currently searched directory, i.e. the directory of the file passed to `lint()`. When run from `lint_package()`, this directory can differ for each linted file.
-3.  A linter file in a parent directory of the currently-searched directory, starting from the deepest path, moving upwards one level at a time.
-4.  A linter file in the user's `HOME` directory.
+3.  A linter file in the `.github/linters` child directory of the currently-searched directory.
+4.  A linter file in a parent directory of the currently-searched directory, starting from the deepest path, moving upwards one level at a time.
```

Comment:
This could be made more clear, stating that 2. and 3. are searched from the parent.

Reply:
Aren't we checking 2. and 3. for the current directory and only if none match, the parent is searched recursively?

This description suggests we first search for `.lintr` in any parent and only look for `.github/linters/.lintr` if there is no `.lintr` anywhere in the path.

Reply:
@klmr this is the only blocker I see. Good to merge once the docs match the behavior.

Reply:
Updated. Please take a look. I think we won’t get around making nested bullet points, since that reflects how the algorithm works. Is this clear enough?